database/worker.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package database
+
+import (
+	"database/sql"
+
+	"github.com/go-vela/types/library"
+)
+
+// Worker is the database representation of a worker.
+type Worker struct {
+	ID            sql.NullInt64  `sql:"id"`
+	Hostname      sql.NullString `sql:"hostname"`
+	Path          sql.NullString `sql:"path"`
+	Online        sql.NullBool   `sql:"online"`
```

Comment:
Could we change this to `Active` instead of `Online`?

`Active` fits better with some of the field names we already have in other types 👍 

https://github.com/go-vela/types/blob/3e7f85be5d18e6fe39e992946a2b7eaa903800d7/database/repo.go#L55

https://github.com/go-vela/types/blob/3e7f85be5d18e6fe39e992946a2b7eaa903800d7/database/user.go#L50

Reply:
Sure, I think that would be more useful since I can imagine a situation in the future where a worker might be "online" but disabled.